roles/plugins/foreman_discovery/tasks/main.yml

```diff
@@ -4,4 +4,16 @@
   vars:
     plugin_installer_options: >
       --enable-foreman-plugin-discovery
+      --enable-foreman-proxy-plugin-discovery
       {{ foreman_discovery_installer_options }}
+
+# TODO until http://projects.theforeman.org/issues/22479 is fixed
+- name: 'check if FDI tar file exists'
+  stat: path=/var/lib/tftpboot/boot/fdi-image-latest.tar
+  register: fdi_image_tar_file
+- name: 'Install latest FDI image'
```

Comment:
Mind an empty line above this to bring it in line with other task files?

Reply:
I wanted to keep the together as the comment applies to both tasks, should I copy the comment then?

Reply:
Maybe move to a separate file then and include it here? `hacks.yaml` or `workarounds.yaml` ? Will make it clearer to me and allow keeping conventions.

Reply:
I'd prefer to get the bug fixed instead.

Reply:
I see you already sent a PR, if that gets merged, I'm happy to remove this workaround, though I think it shouldn't block this PR if it's not merge shortly